mlrun/feature_store/api.py

```diff
@@ -261,12 +265,36 @@ def ingest(
             raise mlrun.errors.MLRunInvalidArgumentError(
                 "data source was not specified"
             )
+
+        if source.schedule:
+            featureset.reload(update_spec=False)
+            if featureset.status.targets and featureset.status.targets[0].last_written:
```

Comment:
last_written will be set only on parquet targets? 
if so, then if the first target is nosql and the second in parquet you won't go into the if statement at all